Edit introductory section of user guide (#139)
AY2324S2-CS2103T-T13-2/tp: docs/UserGuide.md

  pageNav: 3
---

# :tada: InternBook User Guide :books:

## Welcome to InternBook!
Hey there, future interns! Since you are here, we summed that you are also a student, like us, who is starting your internship search. 
Welcome to InternBook, your go-to companion in the exciting journey of applying for internships. 

### Who Are We?
We are a dynamic team of five students who have been in your shoes, feeling the struggle of juggling multiple internship
applications while trying to keep track of every detail. We get it - those Excel spreadsheets might seem like a good idea 
at first, but let's face it, they quickly become a cluttered mess, making it harder to find what you need and leading to 
inconsistencies in your data. That's why we decided to roll up our sleeves and create InternBook!

### What is InternBook?
InternBook isn't just another app - it's your personal assistant for managing internship applications effortlessly. With 

> InternBook, you can record crucial details like company names, application deadlines, contact emails, and the roles you

Reviewer: These things we can put in pointers

are applying for, all in one organised place. It is optimised for a _Command line Interface_ (CLI), but you can use the
advantages of a _Graphical User Interface_ (GUI). No more frantic searches through endless spreadsheets or sticky notes!

### Why Choose InternBook?
- **Simplicity:** Designed for students, whether you have a technical background or not. InternBook is easy to use and follow, 
ensuring a smooth experience for everyone. Say goodbye to complicated Excel templates. InternBook offers a clean and 
intuitive designed with your needs in mind.
- **Organisation:** Keep your internship applications neat and tidy. No more digging through clutter - everything you 
need is right at your fingertips.

> - **Consistency:** Say hello to reliable data. With InternBook, there are error checking for inputs to ensure that the 

Reviewer: ```suggestion
- **Consistency:** Say hello to reliable data. With InternBook, there is error checking for inputs to ensure that the 
```

# InternBook User Guide

## Welcome to InternBook!
Hey there, future interns! Since you are here, we summed that you are also a student, like us, who is starting your internship search. 

> Welcome to InternBook, your go-to companion in the exciting journey of applying for internships. 

Reviewer: I agree with this

applications while trying to keep track of every detail. We get it - those Excel spreadsheets might seem like a good idea 
at first, but let's face it, they quickly become a cluttered mess, making it harder to find what you need and leading to 
inconsistencies in your data. That's why we decided to roll up our sleeves and create InternBook!

### What is InternBook?
InternBook isn't just another app - it's your personal assistant for managing internship applications effortlessly. With 
InternBook, you can record crucial details like company names, application deadlines, contact emails, and the roles you
are applying for, all in one organised place. It is optimised for a _Command line Interface_ (CLI), but you can use the
advantages of a _Graphical User Interface_ (GUI). No more frantic searches through endless spreadsheets or sticky notes!

### Why Choose InternBook?
- **Simplicity:** Designed for students, whether you have a technical background or not. InternBook is easy to use and follow, 
ensuring a smooth experience for everyone. Say goodbye to complicated Excel templates. InternBook offers a clean and 
intuitive designed with your needs in mind.

> - **Organisation:** Keep your internship applications neat and tidy. No more digging through clutter - everything you 

Reviewer 2: I think this could be cluttered data or maybe clutters

for all users.

**Overview of Main Features:**

- **Application Recording:** Easily input and store details of internship applications, including company names, 
application deadlines, contact emails, and roles.
- **Progress Tracking:** Keep track of the status of each application, from not applied to applied, providing clarity
and organisation throughout the application process.
- **Reminder Functionality:** Set reminders for upcoming application deadlines, ensuring timely submissions and proactive
management of internship opportunities.
- **Sorting Options:**
    - **Alphabetical**: Arrange applications alphabetically by company name for easy reference and quick navigation.
    - **Start Date**: Sort applications based on the start date of the internship application, allowing users to prioritise 
    applications based on upcoming opportunities.
    - **End Date**: Sort applications according to the end date of the internship application, facilitating planning and 
    scheduling by highlighting approaching deadlines.
- **User-Friendly Interface:** With a clean and intuitive design, InternBook offers ease of navigation and accessibility,

> prioritising ser experience and satisfaction.

Reviewer 2: I think there is typo, i think you meant "user"

5. Refer to the [Features](#features) below for details of each command.

--------------------------------------------------------------------------------------------------------------------
## How to use the guide

> - The guide introduces the features of our application and shows you an example on the different commands. Stating which
> are the compulsory fields, in a particular format and constraints to it.

Reviewer: ```suggestion
- The guide introduces the features of our application and shows you examples of the different commands in action. It also provides the field constraints, including permitted values and compulsory or optional parameters.
```

> - After the features section, it consists of the frequency asked questions (FAQ), so if you have any doubts, you might

Reviewer 2: I think you meant "frequently" instead

> - After the features section, it consists of the frequency asked questions (FAQ), so if you have any doubts, you might

Reviewer: ```suggestion
- After the features section, we have a frequently asked questions (FAQ) section, so if you have any doubts, you might
```

want to refer to the FAQ to see whether your questions can be answered.

> - After the FAQ section, it consists of known issues, which may contain bugs that we, the developers, are working on. 

Reviewer: ```suggestion
- After the FAQ section, we have an issue tracker which may contain bugs that we, the developers, are working on. 
```

Please be patient with us while we fix the bugs!

> - The last section consist of the command summary where a summary for the commands are listed in a table for better and 

Reviewer: ```suggestion
- The last section consists of the command summary where a summary for every command is listed in a table for better and 
```
